device-types/Cisco/N540-12Z20G-SYS-D.yaml

```diff
@@ -0,0 +1,86 @@
+---
+manufacturer: Cisco
+model: N540-12Z20G-SYS-D
+slug: cisco-n540-12z20g-sys-d
+part_number: N540-12Z20G-SYS-D
+u_height: 1
+is_full_depth: false
+weight: 5.9
+weight_unit: kg
+airflow: left-to-right
+module-bays:
+  - name: PM0
+    position: 0/PM0
+    description: Power module
```

Comment:
Hi Thank you for your contribution,

This looks like it has a fixed power supply if so this should be added under `power-ports`